3_javascript/javascript1/week2/scripts.js

```diff
@@ -0,0 +1,173 @@
+/// Flight booking fullname function /////
+
+function getFullname(firstname, surname) {
+  return firstname + ' ' + surname;
+}
+
+/// Create two variables and assign them the results of the function
+const fullname1 = getFullname('Benjamin', 'Hughes');
+const fullname2 = getFullname('Jaden', 'Smith');
+
+/// Log out the two fullname variables//////
+console.log(fullname1);
+console.log(fullname2);
+
+/// Formal fullname////
+function getFullname(firstname, surname, useFormalName = false) {
+  if (useFormalName === true) {
+    return `Lord ${firstName} ${lastName}`;
+  } else {
+    return `${firstName} ${lastName}`;
+  }
+}
+///Create an extra parameter useFormalName that is a boolean////
+
+const formalFullName = getFullName('peter', 'Dadson', true); // Adds "Lord" in front
+const informalFullName = getFullName('Jane', 'matic', false); // Doesn't add "Lord"
+
+/// Describe how you would fix the getFullname so it also works for women///
+function getFullname(firstName, lastName, gender) {
+  if (gender === 'male') {
+    return 'Mr. ' + firstName + ' ' + lastName;
+  } else if (gender === 'female') {
+    return 'Ms. ' + firstName + ' ' + lastName;
+  } else {
+    return firstName + ' ' + lastName;
+  }
+}
+
+///// Event Application /////
+const weekDay = [
+  'Sunday',
+  'Monday',
+  'Tuesday',
+  'Wednesday',
+  'Thursday',
+  'Friday',
+  'Saturday',
+];
+
+///// Get the current day as a string ////
+const today = weekDay[new Date().getDay()];
+
+const daysToAdd = 5; // Example: event is 5 days from today
+
+// result //
+let eventDay = '';
+
+function eventCalendar(today, daysToAdd) {
+  for (let i = 0; i < weekDay.length; i++) {
+    const day = weekDay[i];
+    if (day === today) {
+      eventDay = weekDay[(i + daysToAdd) % weekDay.length];
+      console.log(
```

Comment:
Here you should return every day. Currently, you have an if condition only returning today.

Reply:
Done with the Changes.